codingTest_김준우/두 개 뽑아서 하기.js

```diff
@@ -0,0 +1,19 @@
+function solution(numbers) {
+  let Arr = [];
```

Comment:
```suggestion
  const arr = [];
```
배열, 객체는 상수로 선언해도 안에있는 값 변경이 가능합니다.

우리 집 안에 가구들이 많이 있는데, 그 가구들을 바꾼다 해서 집이 안바뀌는 거라고 생각하시면 편합니다.
그리고 변수 선언시에는 ```Arr```보다는 ```arr```로하는게 적절합니다. 대문자로 선언하게 되면, 변수가 아닌 다른 값으로 인식될 여지가 있습니다.